org_fedora_oscap/rule_handling.py

```diff
@@ -344,10 +344,10 @@ def eval_rules(self, ksdata, storage, report_only=False):
 
         messages = []
         if self._mount_point not in storage.mountpoints:
-            msg = _("%s must be on a separate partition or logical "
+            msg = _("{0} must be on a separate partition or logical "
                     "volume and has to be created in the "
-                    "partitioning layout before selecting a "
-                    "security profile." % self._mount_point)
+                    "partitioning layout before installation can occur "
+                    "with a security profile".format(self._mount_point)
```

Comment:
Missing closing bracket.

Reply:
Thanks @dahaic. Fixed the missing closing bracket that I stupidly missed.

Reply:
That's what review is for :)